src/folio_migration_tools/mapping_file_transformation/notes_mapper.py

```diff
@@ -69,22 +70,7 @@ def map_notes(self, legacy_object, legacy_id, object_uuid: str, record_type: FOL
                     )
 
     def get_prop(self, legacy_item, folio_prop_name, index_or_id):
-        legacy_item_keys = self.mapped_from_legacy_data.get(folio_prop_name, [])
-
-        if folio_prop_name in self.mapped_from_values and len(legacy_item_keys) == 1:
-            return self.mapped_from_values.get(folio_prop_name, "")
-
-        map_entries = list(
-            MappingFileMapperBase.get_map_entries_by_folio_prop_name(
-                folio_prop_name, self.record_map["data"]
-            )
-        )
-        if len(map_entries) > 1:
-            self.migration_report.add(Blurbs.Details, f"{legacy_item_keys} were concatenated")
-        return " ".join(
-            MappingFileMapperBase.get_legacy_value(legacy_item, map_entry, self.migration_report)
-            for map_entry in map_entries
-        ).strip()
+        return super().get_prop(legacy_item, folio_prop_name, index_or_id)
```

Comment:
Same question as in the previous file. Why is this method defined if all it does is call the same method on the parent class? (admit, I could be missing something…)

Reply:
Equally good point. I cleaned the previous implementation out before, and the logical thing to do is to remove them entirely